nginx-build/build.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```

Comment:
Why do you need this script? Local dev? Maybe the Makefile is a better place for this?

Reply:
To build the nginx inside the Dockerfile.